index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Imperial Donuts</title>
+</head>
+<body>
+    <header>
+        
+        <nav class="nav-bar">
+            <a href="About">About</a>
+            <a href="Donuts">Donuts</a>
+            <a href="Stores">Stores</a>
+            <a href="Contact">Contact</a>
+        </nav>
+        <div class="header-div">
+            <h1>IMPERIAL DONUTS</h1>
+            <h2>Delicious,Scrumptious and full of goodness!</h2>
+        </div>
+    </header>
+    <main>
+        <section class="about-section">About</section>
+        <section class="donuts-section">Donuts</section>
+        <section class="stores-section">Stores</section>
+
+    </main>
+    <footer>
+        <h2>Contacts</h2>
+       <!--<h1>Directory</h1>--> 
```

Comment:
please don't pose unused codes as a comment. if the code is irrelevant delete it.

Reply:
- [ ] I suggest you explain why this change is required, that will make it very easy to implement the changes because the one who will read it will know the context

Reply:
please don't pose unused codes as a comment. if the code is irrelevant delete it. because unused code makes your project unclean.

Reply:
good job 👍